packages/grafana-prometheus/src/datasource.ts

```diff
@@ -715,7 +726,7 @@ export class PrometheusDatasource
 
     if (this.hasLabelsMatchAPISupport()) {
       return (
-        await this.languageProvider.fetchSeriesValuesWithMatch(options.key, expr, requestId, options.timeRange)
+        await this.languageProvider.fetchSeriesValuesWithMatch(options.timeRange, options.key, expr, requestId)
```

Comment:
Same here, changing the parameters can cause some downstream issues for any folks who use the library. 🙏🏻